### PR TITLE
Adding initial commits of ps/2 converter

### PR DESCRIPTION
**Pull request by lbibass**
## Description

mainly just a copying and pasting of the files from the tmk ps/2 converter. Removed a lot of what I think is the excess for simplification. Not sure how to get it to work at this point. Not sure how to fix the `Compiling: keyboards/converter/ps2_usb/matrix.c                                                    keyboards/converter/ps2_usb/matrix.c: In function 'matrix_scan':
keyboards/converter/ps2_usb/matrix.c:217:25: error: implicit declaration of function 'matrix_clear'; did you mean 'matrix_break'? [-Werror=implicit-function-declaration]` error. 

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/converter/ps2_usb/config.h [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+
+
+/* matrix size */
+#define MATRIX_ROWS 32  // keycode bit: 3-0
+#define MATRIX_COLS 8   // keycode bit: 6-4
+
+
+/* key combination for command */
+#define IS_COMMAND() ( \
+    keyboard_report->mods == (MOD_BIT(KC_LSHIFT) | MOD_BIT(KC_RSHIFT)) || \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/config.h]:
```suggestion
    get_mods() == (MOD_MASK_SHIFT) || \
```

File: keyboards/converter/ps2_usb/config.h [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+    keyboard_report->mods == (MOD_BIT(KC_LALT) | MOD_BIT(KC_RALT)) \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/config.h]:
```suggestion
    get_mods() == (MOD_MASK_ALT)) \
```

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    ESC, F1,  F2,  F3,  F4,  F5,  F6,  F7,  F8,  F9,  F10, F11, F12,           PSCR,SLCK,BRK,
+    GRV, 1,   2,   3,   4,   5,   6,   7,   8,   9,   0,   MINS,EQL, BSPC,     INS, HOME,PGUP,    NLCK,PSLS,PAST,PMNS,
+    TAB, Q,   W,   E,   R,   T,   Y,   U,   I,   O,   P,   LBRC,RBRC,BSLS,     DEL, END, PGDN,    P7,  P8,  P9,
+    CAPS,A,   S,   D,   F,   G,   H,   J,   K,   L,   SCLN,QUOT,     ENT,                         P4,  P5,  P6,  PPLS,
+    LSFT,Z,   X,   C,   V,   B,   N,   M,   COMM,DOT, SLSH,          RSFT,          UP,           P1,  P2,  P3,
+    LCTL,LGUI,LALT,          SPC,                     RALT,RGUI,APP, RCTL,     LEFT,DOWN,RGHT,    P0,       PDOT,PENT
+    ),
+};
+
+const action_t PROGMEM fn_actions[] = {
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+};
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]:
```suggestion
```

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+     * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]|    \| |Del|End|PgD| |  7|  8|  9|   |
+     * |-----------------------------------------------------------| `-----------' |-----------|  +|
+     * |CapsLo|  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|Return  |               |  4|  5|  6|   |
+     * |-----------------------------------------------------------|     ,---.     |---------------|
+     * |Shift   |  Z|  X|  C|  V|  B|  N|  M|  ,|  ,|  /|Shift     |     |Up |     |  1|  2|  3|   |
+     * |-----------------------------------------------------------| ,-----------. |-----------|Ent|
+     * |Ctrl |Gui |Alt |         Space         |Alt |Gui |Menu|Ctrl| |Lef|Dow|Rig| |      0|  .|   |
+     * `-----------------------------------------------------------' `-----------' `---------------'
+     */
+    KEYMAP(
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]:
```suggestion
    LAYOUT_fullsize_ansi(
```

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+    ESC, F1,  F2,  F3,  F4,  F5,  F6,  F7,  F8,  F9,  F10, F11, F12,           PSCR,SLCK,BRK,
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]: Please change all of these keycodes to include the `KC_` prefix.

File: keyboards/converter/ps2_usb/led.c [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#include "stdint.h"
+#include "ps2.h"
+#include "led.h"
+
+
+void led_set(uint8_t usb_led)
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/led.c]: Please move this function into ps2_usb.c as `led_set_kb()`.

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 8]
```diff
@@ -0,0 +1,189 @@
...
+                                                                                                \
+    SYSTEM_POWER, SYSTEM_SLEEP, SYSTEM_WAKE,                                                    \
+    MEDIA_NEXT_TRACK, MEDIA_PREV_TRACK, MEDIA_STOP, MEDIA_PLAY_PAUSE, MEDIA_SELECT,             \
+    MAIL, CALCULATOR, MY_COMPUTER,                                                              \
+    WWW_SEARCH, WWW_HOME, WWW_BACK, WWW_FORWARD,                                                \
+    WWW_STOP, WWW_REFRESH, WWW_FAVORITES                                                        \
+)
+
+/* US layout */
+#define LAYOUT( \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
#define LAYOUT_fullsize_ansi( \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 9]
```diff
@@ -0,0 +1,189 @@
...
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,  VOLD,VOLU,MUTE,  \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,JPY,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,    K5D, KF1,KE9,KFA,  K6C,K75,K7D,K79, \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,        K5A,               K6B,K73,K74,PCMM,\
+    K12,NUBS,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,   RO, K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11,MHEN,   K29,          HENK,KANA,K91,KA7,KAF,K94, KEB,KF2,KF4,  PWR,K70,K71,PEQL \
+)
+
+/* ISO layout */
+#define LAYOUT_ISO( \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
#define LAYOUT_fullsize_iso( \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 10]
```diff
@@ -0,0 +1,189 @@
...
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,  VOLD,VOLU,MUTE,  \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,JPY,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,    K5D, KF1,KE9,KFA,  K6C,K75,K7D,K79, \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,        K5A,               K6B,K73,K74,PCMM,\
+    K12,K61,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,    RO, K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11,MHEN,   K29,          HENK,KANA,K91,KA7,KAF,K94, KEB,KF2,KF4,  PWR,K70,K71,PEQL \
+)
+
+/* JIS layout */
+#define LAYOUT_JIS( \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
#define LAYOUT_fullsize_jis( \
```

**Comment by fauxpark** [comment]
Also, could you revert the addition of all the .ipch files in the .vscode directory?

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 11]
```diff
@@ -0,0 +1,189 @@
...
+    K12,K61,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,    K51,K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11,K67,    K29,            K64,K13,K91,KA7,KAF,K94, KEB,KF2,KF4,  K68,K70,K71,K63, \
+                                                                                            \
+    KB7, KBF, KDE,           /* System Power, Sleep, Wake */                                \
+    KCD, K95, KBB, KB4, KD0, /* Next, Previous, Stop, Pause, Media Select */                \
+    KC8, KAB, KC0,           /* Mail, Calculator, My Computer */                            \
+    K90, KBA, KB8, KB0,      /* WWW Search, Home, Back, Forward */                          \
+    KA8, KA0, K98            /* WWW Stop, Refresh, Favorites */                             \
+) { \
+    { KC_NO,    KC_##K01, KC_NO,    KC_##K03, KC_##K04, KC_##K05, KC_##K06, KC_##K07 }, \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/ps2_usb.h]: Please remove the `KC_##` prefixes here.

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 12]
```diff
@@ -0,0 +1,189 @@
...
+ * |-----------------------------------------------------------|     ,---.     |---------------|
+ * | 12 | 61| 1A| 22| 21| 2A| 32| 31| 3A| 41| 49| 4A| 51|  59  |     | F5|     | 69| 72| 7A| DA|
+ * |-----------------------------------------------------------| ,-----------. |---------------|
+ * | 14| 9F| 11| 67 |     29         | 64 | 13 | 91| A7| AF| 94| | EB| F2| F4| | 68|70 | 71| 63|
+ * `-----------------------------------------------------------' `-----------' `---------------'
+ * ^a ISO hash key uses identical scancode 5D to US backslash.
+ * 51, 63, 68, 6D: hidden keys in IBM model M
+ */
+/* All keys */
+#define LAYOUT_ALL( \
```
Comment (noroadsleft) [inline keyboards/converter/ps2_usb/ps2_usb.h]: QMK custom is that layout macro names begin with `LAYOUT`, and from then on are fully lowercase.

This should be renamed.

```suggestion
#define LAYOUT_all( \
```

File: keyboards/converter/ps2_usb/README.md [hunk 13]
```diff
@@ -0,0 +1,78 @@
+PS/2 to USB keyboard converter
```
Comment (noroadsleft) [inline keyboards/converter/ps2_usb/README.md]: Could you please rename this file as `readme.md`?

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 14]
```diff
@@ -0,0 +1,189 @@
...
+#define LAYOUT_FULL( \
+        K08,K10,K18,K20,K28,K30,K38,K40,K48,K50,K57,K5F,                                        \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,      KA1,KB2,KA3, \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K6A,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,    K5D, KF1,KE9,KFA,  K6C,K75,K7D,K79, \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,        K5A,               K6B,K73,K74,K6D, \
+    K12,K61,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,    K51,K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11,K67,    K29,            K64,K13,K91,KA7,KAF,K94, KEB,KF2,KF4,  K68,K70,K71,K63  \
+) \
+LAYOUT_ALL( \
```
Comment (noroadsleft) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
LAYOUT_all( \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 15]
```diff
@@ -0,0 +1,189 @@
...
+/* US layout */
+#define LAYOUT( \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,     KFC,K7E,KFE,                   \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,K5D, KF1,KE9,KFA,  K6C,K75,K7D,     \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,    K5A,               K6B,K73,K74,K79, \
+    K12,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,        K59,     KF5,      K69,K72,K7A,     \
+    K14,K9F,K11,        K29,                K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71,KDA  \
+) \
+LAYOUT_FULL( \
```
Comment (noroadsleft) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
LAYOUT_full( \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 16]
```diff
@@ -0,0 +1,189 @@
...
+/* ISO layout */
+#define LAYOUT_ISO( \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,     KFC,K7E,KFE,                   \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,     KF1,KE9,KFA,  K6C,K75,K7D,     \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,K5D,K5A,               K6B,K73,K74,K79, \
+    K12,K61,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,    K59,     KF5,      K69,K72,K7A,     \
+    K14,K9F,K11,        K29,                K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71,KDA  \
+) \
+LAYOUT_FULL( \
```
Comment (noroadsleft) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
LAYOUT_full( \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 17]
```diff
@@ -0,0 +1,189 @@
...
+/* JIS layout */
+#define LAYOUT_JIS( \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,                   \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K6A,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,         KF1,KE9,KFA,  K6C,K75,K7D,     \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,K5D,    K5A,               K6B,K73,K74,K79, \
+    K12,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,K51,        K59,     KF5,      K69,K72,K7A,     \
+    K14,K9F,K11,    K67,K29,K64,K13,            K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71,KDA  \
+) \
+LAYOUT_FULL( \
```
Comment (noroadsleft) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
LAYOUT_full( \
```

File: keyboards/converter/ps2_usb/matrix.c [hunk 18]
```diff
@@ -0,0 +1,446 @@
...
+    }
+
+    // TODO: request RESEND when error occurs?
+/*
+    if (PS2_IS_FAILED(ps2_error)) {
+        uint8_t ret = ps2_host_send(PS2_RESEND);
+        xprintf("Resend: %02X\n", ret);
+    }
+*/
+    return 1;
```
Comment (drashna) [inline keyboards/converter/ps2_usb/matrix.c]: Please add `matrix_scan_quantum` here. 
```suggestion
    matrix_scan_quantum();
    return 1;
```

File: keyboards/converter/ps2_usb/README.md [hunk 19]
```diff
@@ -0,0 +1,78 @@
...
+Connect Wires
+-------------
+1. Connect **Vcc** and **GND**.
+2. Connect **Clock** and **Data** line. 
+    - **Interrupt**:   **Clock** is on `PD1` and **Data** on `PD0`.(Recommended. Soarer's converter compatible)
+    - **Busywait**:    **Clock** is on `PD1` and **Data** on `PD0`.
+    - **USART**:       **Clock** is on `PD5` and **Data** on `PD2`.
+3. You need pull-up resistors. 1K-10K Ohm would be fine.
+
+To change pin configuration edit **config.h** and **Makefile**.
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
To change pin configuration edit **config.h** and **rules.mk**.
```

File: keyboards/converter/ps2_usb/README.md [hunk 20]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Build Firmware
+--------------
+For **TMK converter Rev.2**:
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
For **QMK converter**:
```

File: keyboards/converter/ps2_usb/README.md [hunk 21]
```diff
@@ -0,0 +1,78 @@
...
+
+    $ make -f Makefile.rev2 clean
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]: This isn't needed, technically
```suggestion
```

File: keyboards/converter/ps2_usb/README.md [hunk 22]
```diff
@@ -0,0 +1,78 @@
...
+    $ make -f Makefile.rev2 KEYMAP=plain
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
    $ make converter/ps2_usb:default
```

Or even:
```suggestion
    $ make distclean converter/ps2_usb:default
```

File: keyboards/converter/ps2_usb/README.md [hunk 23]
```diff
@@ -0,0 +1,78 @@
...
+
+To program firmware push the button on converter and run:
+
+    $ make -f Makefile.rev2 KEYMAP=plain dfu
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
    $ make converter/ps2_usb:default:dfu
```

but if this is a teensy controller (eg, halfkay), then it should be:
```suggestion
    $ make converter/ps2_usb:default:teensy
```

File: keyboards/converter/ps2_usb/README.md [hunk 24]
```diff
@@ -0,0 +1,78 @@
...
+
+
+- For **TMK converter Rev.1** use `make -f Makefile.rev1` instead.
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
```

File: keyboards/converter/ps2_usb/README.md [hunk 25]
```diff
@@ -0,0 +1,78 @@
...
+- To select keymap use `jis`, `spacefn` or your own in place of `plain`.
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
```

File: keyboards/converter/ps2_usb/README.md [hunk 26]
```diff
@@ -0,0 +1,78 @@
...
+
+
+Keymap
+------
+Several version of keymap are available in advance but you are recommended to define your favorite layout yourself. To define your own keymap create file named `keymap_<name>.c` and see keymap document(you can find in README.md of top directory) and existent keymap files.
```
Comment (drashna) [inline keyboards/converter/ps2_usb/README.md]:
```suggestion
Several version of keymap are available in advance but you are recommended to define your favorite layout yourself. To define your own keymap create a folder with your github username, and in it, a file named `keymap.c` and see keymap document(you can find in README.md of top directory) and existent keymap files.
```

File: keyboards/converter/ps2_usb/config.h [hunk 27]
```diff
@@ -0,0 +1,191 @@
...
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x6512
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    t.m.k.
```
Comment (drashna) [inline keyboards/converter/ps2_usb/config.h]:
```suggestion
#define MANUFACTURER    QMK Community
```

**Comment by drashna** [comment]
Also, could you remove the changes to .vscode?

File: keyboards/converter/ps2_usb/matrix.c [hunk 28]
```diff
@@ -0,0 +1,446 @@
...
+#include "util.h"
+#include "debug.h"
+#include "ps2.h"
+#include "host.h"
+#include "led.h"
+#include "matrix.h"
+
+static void matrix_make(uint8_t code);
+static void matrix_break(uint8_t code);
+
```
Comment (drashna) [inline keyboards/converter/ps2_usb/matrix.c]: To fix at least one of the compilation errors, move `matrix_clear` higher up in the file, or add this here: 
```suggestion
void matrix_clear(void);

```

File: keyboards/converter/ps2_usb/ps2_usb.h (hunk 11 above)
Comment (lbibass) [reply keyboards/converter/ps2_usb/ps2_usb.h]: will do.

File: keyboards/converter/ps2_usb/README.md (hunk 13 above)
Comment (lbibass) [reply keyboards/converter/ps2_usb/README.md]: yup

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 29]
```diff
@@ -0,0 +1,190 @@
...
+)
+
+/* JIS layout */
+#define LAYOUT_fullsize_jis( \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,                   \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K6A,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,         KF1,KE9,KFA,  K6C,K75,K7D,     \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,K5D,    K5A,               K6B,K73,K74,K79, \
+    K12,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,K51,        K59,     KF5,      K69,K72,K7A,     \
+    K14,K9F,K11,    K67,K29,K64,K13,            K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71, KDA,\
```
Comment (drashna) [inline keyboards/converter/ps2_usb/ps2_usb.h]: No need for trailing comma. In fact, it's causing an error. 

```suggestion
    K14,K9F,K11,    K67,K29,K64,K13,            K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71, KDA \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 30]
```diff
@@ -0,0 +1,190 @@
...
+) \
+
+LAYOUT_full( \
+                                              \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,    \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K6A,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,    K5D, KF1,KE9,KFA,  K6C,K75,K7D,K79, \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,        K5A,               K6B,K73,K74,\
+    K12, K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,   K51,K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11,K67,    K29,            K64,K13,K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,K71, \
```
Comment (drashna) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
    K14,K9F,K11,K67,    K29,            K64,K13,K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,K71 \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 31]
```diff
@@ -0,0 +1,190 @@
...
+    K14,K9F,K11,        K29,                K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71,KDA  \
+) \
+LAYOUT_full( \
+                                              \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,    \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,    K5D, KF1,KE9,KFA,  K6C,K75,K7D,K79, \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,        K5A,               K6B,K73,K74,\
+    K12, K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,   , K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11   K29,          K91,KA7,KAF,K94, KEB,KF2,KF4,  ,K70,K71, \
```
Comment (drashna) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
    K14,K9F,K11   K29,          K91,KA7,KAF,K94, KEB,KF2,KF4,  ,K70,K71 \
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 32]
```diff
@@ -0,0 +1,190 @@
...
+    K14,K9F,K11,        K29,                K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71,KDA  \
+) \
+LAYOUT_full( \
+                                              \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,         KFC,K7E,KFE,    \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,    K5D, KF1,KE9,KFA,  K6C,K75,K7D,K79, \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,        K5A,               K6B,K73,K74,\
+    K12,K61,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,    , K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11   K29,          K91,KA7,KAF,K94, KEB,KF2,KF4,  ,K70,K71, \
```
Comment (drashna) [inline keyboards/converter/ps2_usb/ps2_usb.h]:
```suggestion
    K14,K9F,K11   K29,          K91,KA7,KAF,K94, KEB,KF2,KF4,  ,K70,K71 \
```

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 33]
```diff
@@ -0,0 +1,30 @@
+#include "ps2_usb.h"
+
+
+const uint8_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```
Comment (drashna) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]:
```suggestion
const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
```

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 34]
```diff
@@ -0,0 +1,190 @@
...
+/* US layout */
+#define LAYOUT_fullsize_ansi( \
+    K76,K05,K06,K04,K0C,K03,K0B,K83,K0A,K01,K09,K78,K07,     KFC,K7E,KFE,                   \
+    K0E,K16,K1E,K26,K25,K2E,K36,K3D,K3E,K46,K45,K4E,K55,K66, KF0,KEC,KFD,  K77,KCA,K7C,K7B, \
+    K0D,K15,K1D,K24,K2D,K2C,K35,K3C,K43,K44,K4D,K54,K5B,K5D, KF1,KE9,KFA,  K6C,K75,K7D,     \
+    K58,K1C,K1B,K23,K2B,K34,K33,K3B,K42,K4B,K4C,K52,    K5A,               K6B,K73,K74,K79, \
+    K12,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,        K59,     KF5,      K69,K72,K7A,     \
+    K14,K9F,K11,        K29,                K91,KA7,KAF,K94, KEB,KF2,KF4,  K70,    K71,KDA  \
+) \
+LAYOUT_full( \
```
Comment (drashna) [inline keyboards/converter/ps2_usb/ps2_usb.h]: This part is wrong. It matches what is above, .... which is the problem. It needs to match the top part of `LAYOUT_full` instead.

File: keyboards/converter/ps2_usb/ps2_usb.h [hunk 35]
```diff
@@ -0,0 +1,190 @@
...
+    K12,K61,K1A,K22,K21,K2A,K32,K31,K3A,K41,K49,K4A,    K51,K59,     KF5,      K69,K72,K7A,KDA, \
+    K14,K9F,K11,K67,    K29,            K64,K13,K91,KA7,KAF,K94, KEB,KF2,KF4,  K68,K70,K71,K63, \
+                                                                                            \
+    KB7, KBF, KDE,           /* System Power, Sleep, Wake */                                \
+    KCD, K95, KBB, KB4, KD0, /* Next, Previous, Stop, Pause, Media Select */                \
+    KC8, KAB, KC0,           /* Mail, Calculator, My Computer */                            \
+    K90, KBA, KB8, KB0,      /* WWW Search, Home, Back, Forward */                          \
+    KA8, KA0, K98            /* WWW Stop, Refresh, Favorites */                             \
+) { \
+    { KC_NO,    KC_K01, KC_NO,    KC_K03, KC_K04, KC_K05, KC_K06, KC_K07 }, \
```
Comment (fauxpark) [inline keyboards/converter/ps2_usb/ps2_usb.h]: Please remove the `KC_` prefix on the entries here that aren't `KC_NO`.

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 36]
```diff
@@ -0,0 +1,30 @@
+#include "ps2_usb.h"
```
Comment (drashna) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]: This needs to be changed 
```suggestion
#include QMK_KEYBOARD_H
```

File: keyboards/converter/ps2_usb/keymaps/default/keymap.c [hunk 37]
```diff
@@ -0,0 +1,30 @@
...
+#include "ps2.h"
```
Comment (drashna) [inline keyboards/converter/ps2_usb/keymaps/default/keymap.c]: This should be moved. It doesn't belong here.

```suggestion
```

File: keyboards/converter/ps2_usb/rules.mk [hunk 38]
```diff
@@ -0,0 +1,58 @@
...
+#   comment out to disable the options.
+#
+BOOTMAGIC_ENABLE = no   # Virtual DIP switch configuration(+1000)
+MOUSEKEY_ENABLE  = yes  # Mouse keys(+4700)
+EXTRAKEY_ENABLE  = yes  # Audio control and System control(+450)
+CONSOLE_ENABLE   = yes  # Console for debug(+400)
+COMMAND_ENABLE   = yes  # Commands for debug and configuration
+NKRO_ENABLE      = yes  # USB Nkey Rollover
+CUSTOM_MATRIX    = yes
+
```
Comment (drashna) [inline keyboards/converter/ps2_usb/rules.mk]: I think you need to add: 

```suggestion
PS2_USE_USART = yes

```

File: keyboards/converter/ps2_usb/README.md (hunk 13 above)
Comment (noroadsleft) [reply keyboards/converter/ps2_usb/README.md]: This file still needs to be renamed.

File: keyboards/converter/ps2_usb/led.c (hunk 7 above)
Comment (lbibass) [reply keyboards/converter/ps2_usb/led.c]: I think I did it correctly? I am confused as to what to do.

**Pull request closed without merging**